### PR TITLE
refactor: Ticketing tab screens according to new guidelines

### PR DESCRIPTION
**Pull request by gorandalum**
### Background
As always, not possible to do this without touching lots of files. Hopefully this is the last screens that creates such a big PR. As far as I can see, after this, the remaining screens to refactor are:
- login screens
- fare contract screens

After this has been done, I think we can start with looking at eslint rules for enforcing the guidelines. 

Note: Some changes are just import changes. I ran optimize imports on some folders.

### Acceptance criteria
Everything should work as before. Especially ticketing logic. This includes:
- [ ] Ticketing tabs
- [ ] Ticket history in Profile
- [ ] Ticket integration (purchase and viewing purchased) from dashboard
- [ ] Redirecting to active tickets tab after ticket purchase.

### CODE REVIEW

File: src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/index.tsx
Comment (rosvik) [inline src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/index.tsx:1]: This can be renamed to index.ts

**Review by rosvik — APPROVED** [review]
Nice! LGTM

**Review by jorelosorio — APPROVED** [review]
As @gorandalum mentioned it might be hard to go line by line but just by taking a look it's more renaming, moving code from one place to another, I did not see in a search that there were code changes on it. So I am also approving it.

**Comment by HanneLH** [comment]
Android V.1.35 (1680161690) og iOS V.1.35 (1680161686) 
Testing:
- [ ] Singel ticket as anonym 
- [ ] Singel ticket with profile 
- [ ] Buy with
    - [ ] Vipps
    - [ ] New MC
    - [ ] New Card
    - [ ] Saved MC
    - [ ] Saved Card
- [ ] Periodic ticket 
- [ ] 24 hour ticket 
- [ ] Repeat purchase 
- [ ] Flow from buy to active ticket page

**Pull request merged**